Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -61,6 +61,12 @@
     },
     "%lld %@" : {
       "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : ""
+          }
+        },
```

Comment:
I think this shouldn't be empty, but should be `%lld %@` because this is filled in automatically, from computed properties and such, if it was empty, I think it would end up showing as an empty string, when there should be a value there